testing/steps/test_column_transformer.py

```diff
@@ -363,18 +372,18 @@ def test_column_transformer_fit_transform_should_support_multiple_tuples():
         n_dimension=3
     )
     data_inputs = test_case.data_inputs
-    column_transformer = ColumnTransformer(test_case.column_transformer_tuple_list)
+    p = Pipeline([ColumnTransformer(test_case.column_transformer_tuple_list)])
```

Comment:
Why `Pipeline([ColumnTransformer` ?? I hope we can use `ColumnTransformer` without wrapping it by pipeline. Maybe use the ForceHandleMixin as I proposed above then.